Sources/Class-Package.php

```diff
@@ -793,13 +793,15 @@ public function connect($ftp_server, $ftp_port = 21, $ftp_user = 'anonymous', $f
 		if (!$this->connection)
 		{
 			$this->error = 'bad_server';
+            $this->last_message = 'Invalid Server';
```

Comment:
Perhaps your editor is automagically changing tabs for spaces or viceversa when saving?